andela/trime
# [Fixes #121560565] Fix showing wrong trimes

### What this PR does
Fixes bug where if you are tracking time in one project, then pause that and create
another project and then start tracking time in the new project, the
project page will show time tracks for the previously tracked project. This issue would resolve if you reloaded the page.

### How can this be manually tested
Create a project in an organisation and start a time track in that project. Leave that project's time track running or paused, then go back and create another project. When you start tracking time in the new project, you should only see the time tracks in that project.

### Relevant Pivotal Tracker story
[Project page shows time tracks for another project.](https://www.pivotaltracker.com/story/show/121560565)
